ui/inspectors/tunable.reel/tunable.js

```diff
@@ -10,18 +10,18 @@ var Component = require("montage/ui/component").Component,
  * @extends Component
  */
 exports.Tunable = Component.specialize(/** @lends Tunable# */ {
-    typeOPTIONS: {
+    typeOptions: {
         value: null
     },
 
     enterDocument: {
         value: function(isFirstTime) {
             var self = this,
-                tunableType = TunableType;
+                EnumTunableType = TunableType;
             if (isFirstTime) {
-                this.typeOPTIONS = [];
-                for (var i = 0; i < tunableType.members.length; i++) {
-                    this.typeOPTIONS.push({label: tunableType.members[i], value: tunableType[tunableType.members[i]]});
+                this.typeOptions = [];
```

Comment:
You should just assign `TunableType.members` to `this.typeOptions`, no need to iterate by yourself.
Also you should do this in `templateDidLoad` (everything that is run only once and doesn't need to read a bound variable should go in `templateDidLoad`).